articles/native-platforms/ios-swift/_includes/_api_authz.md

```diff
@@ -0,0 +1,131 @@
+## Configuring Your Application
+
+## Dependences
+
+This quickstart uses the Auth0.swift SDK to help you add authentication and API authorization to your iOS app. You will need to add this dependency to your Podfile.
+
+```ruby
+use_frameworks!
+pod "Auth0", :git => 'https://github.com/auth0/Auth0.swift.git'
```

Comment:
Also we can add  instructions about Carthage:

Which is something like :

Add the following to your Cartfile

```
github "auth0/Auth0.swift" ~> 1.0
```

run `carthage bootstrap --platform ios` and add `Auth0.framework` to your application.